modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase

```diff
@@ -74,9 +74,21 @@ definition {
 	@description = "LPS-175645 - It's possible to export Picklist (download)"
 	@priority = 5
 	test CanExportPicklist {
+		task ("Given a Picklist created") {
+			Picklist.addPicklistViaAPI(picklistName = "Test");
+		}
 
-		// TODO pending implementation
+		task ("When a picklist is exported") {
+			Picklist.gotoPicklists();
 
+			Picklist.gotoPicklistExport(
+				picklistName = "Test",
+				picklistOption = "Export");
+		}
+
+		task ("Then Picklist is exporting successfully") {
+			Picklist.assertDownloadedPicklist(downloadedFile = "ListType_*");
```

Comment:
"Then the Picklist is exported successfully"
instead of ListType_*, let's assert the name of the picklist so ListType_{newPicklistName}. Reasoning again is just to be clear the correct picklist got exported.

Reply:
I struggled a bit with the assert since every time a picklist is downloaded, the number that comes after ListType_"{number}" is random, and the usage of "*" was Tim's indication. The numbers that come after ListType is not chosen by the user.